plugins/catalog-backend/src/ingestion/processors/LocationEntityProcessor.ts

```diff
@@ -17,26 +17,39 @@
 import { Entity, LocationEntity, LocationSpec } from '@backstage/catalog-model';
 import * as result from './results';
 import { CatalogProcessor, CatalogProcessorEmit } from './types';
+import path from 'path';
+
+export function toAbsoluteUrl(base: LocationSpec, target: string): string {
+  try {
+    if (base.type === 'file') {
+      if (target.startsWith('.')) {
+        return path.join(path.dirname(base.target), target);
```

Comment:
Prolly wanna check that `base.target` doesn't end with a '/' or equivalent before grabbing the dirname right? I'm thinking that a base location of `/root/data/` with a target of `./catalog-info.yaml` would resolve to `/root/data/catalog-info.yaml`. Not really technically possible to have that happen right now but thinking it makes sense to handle correctly.

Reply:
So to be clear, SHOULD that result in `/root/data/catalog-info.yaml` or not? I'm actually not sure :) I'm usually of the opinion that trailing slashes should have no effect

Reply:
Could throw an error if there's a trailing path separator for now, that way we get to decide later and it's not needed right now anyway :grin:

Reply:
That sounds wise.

Reply:
@Rugvip done!

Reply:
(below)